lib/ansible/modules/utilities/logic/wait_for.py

```diff
@@ -314,28 +314,30 @@ def _get_exclude_ips(self):
     def get_active_connections_count(self):
         active_connections = 0
         for family in self.source_file.keys():
```

Comment:
Can we do `.items()` instead?

Reply:
I tried to to touch initial code with my fix, rewriting this part would require deeper review which i didn't intend to do

Reply:
Since you manually extract the value anyway it makes sense to just put it here.